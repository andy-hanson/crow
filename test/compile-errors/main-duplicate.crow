main nat^(_ string[])
	()

main nat^(_ string[])
	()
