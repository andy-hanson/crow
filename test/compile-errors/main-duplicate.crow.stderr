[1mtest/compile-errors/main-duplicate.crow[31m 4:1-4:2[m module contains multiple valid 'main' functions
